extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -345,6 +347,44 @@ export class AmpA4A extends AMP.BaseElement {
 
     /** @private {string} */
     this.safeframeVersion_ = DEFAULT_SAFEFRAME_VERSION;
+
+    /** @private @const {!./refresh-manager.RefreshManager}  */
+    this.refreshManager_ = refreshManagerFor(this.win);
+
+    this.refreshManager_.registerElement(this.element, refresher => {
```

Comment:
It will be both. As mentioned in the PR description, this isn't implemented yet.